src/renderer/widgets/WalletDetails/ui/ProxiesList.tsx

```diff
@@ -0,0 +1,133 @@
+                <div className="flex items-center gap-x-2">
+                  <HelpText className="text-text-secondary break-all">{proxiedAddress}</HelpText>
+                  <IconButton
+                    className="shrink-0"
+                    name="copy"
+                    size={20}
+                    onClick={() => copyToClipboard(proxiedAddress)}
+                  />
+                </div>
+              </ExplorersPopover.Group>
+            </ExplorersPopover>
+          </DropdownIconButton.Item>
+          {canCreateProxy && (
+            <DropdownIconButton.Item>
+              <DropdownIconButton.Option option={forgetProxyAction} />
+            </DropdownIconButton.Item>
+          )}
+        </DropdownIconButton.Items>
+      </DropdownIconButton>
+    );
+  };
+
+  return (
+    <div className={cnTw('flex flex-col', className)}>
+      <div className="flex items-center px-5 pb-2">
+        <FootnoteText className="flex-1 px-2 text-text-tertiary">{t('accountList.addressColumn')}</FootnoteText>
+      </div>
+
+      <ul className="flex flex-col px-5 divide-y divide-divider overflow-y-auto overflow-x-hidden">
+        {proxyChainGroups.map((chainGroup) => {
+          const { chainId, totalDeposit } = chainGroup;
+
+          const chain = chainsService.getChainById(chainId);
+
+          if (!chain || !proxiesByChain[chainId] || !proxiesByChain[chainId].length) return;
+
+          return (
+            <li key={chainId} className="flex items-center py-2">
+              <Accordion isDefaultOpen>
+                <Accordion.Button buttonClass="p-2 rounded hover:bg-action-background-hover focus:bg-action-background-hover">
+                  <div className="flex gap-x-2 items-center justify-between flex-1 pr-2">
+                    <ChainTitle className="flex-1" fontClass="text-text-primary" chain={chain} />
+                    <HelpText className="text-text-tertiary">
+                      {t('walletDetails.common.proxyDeposit')}
+                      &nbsp;
+                      <AssetBalance
+                        value={totalDeposit}
+                        asset={chain?.assets[0]}
+                        showIcon={false}
+                        className="text-help-text"
+                      />
+                    </HelpText>
+                  </div>
+                </Accordion.Button>
+                <Accordion.Content>
+                  <ul className="flex flex-col gap-y-2">
+                    {(proxiesByChain[chainId] || []).map((proxy) => (
+                      <li className="px-2 py-1.5" key={proxy.accountId}>
+                        <ProxyAccountComponent
+                          accountId={proxy.accountId}
+                          proxyType={proxy.proxyType}
+                          addressPrefix={chain?.addressPrefix}
+                          actionButton={getProxyActionButton(proxy, chain)}
```

Comment:
You can create local component `ProxyAccountWithActions` move there `ProxyAccount` filled by `DropdownButton` actions without an additional function, pure jsx